frontend/src/pages/notebookController/SpawnerPage.tsx

```diff
@@ -103,13 +104,16 @@ const SpawnerPage: React.FC = React.memo(() => {
     if (shouldRedirect) {
       if (notebookLoaded && notebook) {
         if (!isNotebookRunning) {
-          setStartShown(true);
+          // We already notify the user when they are trying to close/refresh the page
+          // when they are spawning a notebook
+          // so we can safely delete it if the notebook is still there
+          deleteNotebook(projectName, notebook.metadata.name);
```

Comment:
This concerns me a bit -- but my server is down atm so I'll have to test this tomorrow.